vsts/scripts/copySdksFromProdToStorageAccount.sh

```diff
@@ -72,15 +72,18 @@ if [ ! -f "$azCopyDir/azcopy" ]; then
     $azCopyDir/azcopy --version
 fi
 
-destinationSdk="$1"
+destinationSdkUrl="https://$1.blob.core.windows.net"
 sasToken=""
 
-if [ "$1" == $SANDBOX_SDK_STORAGE_BASE_URL ]; then
+if [ "$destinationSdkUrl" == $SANDBOX_SDK_STORAGE_BASE_URL ]; then
     sasToken=$SANDBOX_STORAGE_SAS_TOKEN
-elif [ "$1" == $DEV_SDK_STORAGE_BASE_URL ]; then
+elif [ "$destinationSdkUrl" == $DEV_SDK_STORAGE_BASE_URL ]; then
     sasToken=$DEV_STORAGE_SAS_TOKEN
+# check if the peronal sas token has been found in the oryx key vault
+elif [ "$PERSONAL_STORAGE_SAS_TOKEN" != "\$($1-PERSONAL-STORAGE-SAS-TOKEN)" ]; then 
```

Comment:
Would you mind clarifying what we're comparing here with these two values?

Reply:
we are checking to see if PERSONAL_STORAGE_SAS_TOKEN was contains a SAS token or not

in the template, we have
`PERSONAL_STORAGE_SAS_TOKEN: $(${{ parameters.destinationSdkStorageAccountName }}-PERSONAL-STORAGE-SAS-TOKEN)`

lets take `oryxsdkspauldorsch` for example. There exists an oryxsdkspauldorsch-PERSONAL-STORAGE-SAS-TOKEN in the keyvault, so PERSONAL_STORAGE_SAS_TOKEN env var will contain the SAS token.

if we try with `oryxsdksfake`, where there is no oryxsdksfake-PERSONAL-STORAGE-SAS-TOKEN in the keyvault, PERSONAL_STORAGE_SAS_TOKEN env var will just equal the literal string value `"$(oryxsdksfake-PERSONAL-STORAGE-SAS-TOKEN)"`

so this elif statement is checking to see if the SAS token was found in the key vault, or if the PERSONAL_STORAGE_SAS_TOKEN env var was set to the literal string value